From 08ead36132b44ab1a763fdb2f1313f4d2beae0ca Mon Sep 17 00:00:00 2001
From: Shawn Landden <shawnlandden@gmail.com>
Date: Tue, 3 Jul 2012 12:54:54 -0700
Subject: [PATCH 4/6] configure: GNU/kFreeBSD and disable -rpath

use --as-needed on GNU/kFreeBSD as well as Linux

also, disable -rpath on Linux
---
 configure.in |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index c11de95..ba9a775 100644
--- a/configure.in
+++ b/configure.in
@@ -265,9 +265,9 @@ then
 fi
 
 # As-needed linking
-if uname | grep Linux >config.tmp
+if uname -o | egrep "GNU/Linux|GNU/kFreeBSD" >config.tmp
 then
-  MYLDFLAGS="$MYLDFLAGS -Wl,-rpath-link,.:/usr/local/lib:$MYLDLIBPATH"
+#  MYLDFLAGS="$MYLDFLAGS -Wl,-rpath-link,.:/usr/local/lib:$MYLDLIBPATH"
   MYLDFLAGS="$MYLDFLAGS -Wl,--as-needed"
 else
   MYCMDLIBS="$MYCMDLIBS $LIBS"
-- 
1.7.9.5

